hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/HybridCarboniteConfig.scala

```diff
@@ -40,35 +45,39 @@ case object InactiveMetadataDeletionConfig extends MetadataDeletionConfig
-      if (carboniterConfig.hasPath("freeze-scan")) {
-        val freeze = carboniterConfig.getConfig("freeze-scan")
+    def metadataFreezingConfig: Checked[MetadataFreezingConfig] = {
+      if (carboniterConfig.hasPath("metadata-freezing")) {
+        val defaultInitialInterval: Duration = Duration.Inf
+        val defaultMaxInterval = 5 minutes
+        val defaultMultiplier = 1.1
+        val defaultMinimumSummaryEntryId: Option[Long] = None
+        val defaultDebugLogging = true
 
-        val initialInterval = Try { freeze.getOrElse("initial-interval", 5 seconds) } toErrorOr
-        val maxInterval = Try { freeze.getOrElse("max-interval", default = 5 minutes) } toErrorOr
-        val multiplier = Try { freeze.getOrElse("multiplier", 1.1) } toErrorOr
+        val freeze = carboniterConfig.getConfig("metadata-freezing")
 
-        val errorOrFreezeScanConfig = (initialInterval, maxInterval, multiplier) mapN {
-          case (i, m, x) => HybridCarboniteFreezeScanConfig(i, m, x)
-        } contextualizeErrors "parse Carboniter 'freeze-scan' stanza"
+        val initialInterval = Try { freeze.getOrElse("initial-interval", defaultInitialInterval) } toErrorOr
+        val maxInterval = Try { freeze.getOrElse("max-interval", defaultMaxInterval) } toErrorOr
+        val multiplier = Try { freeze.getOrElse("multiplier", defaultMultiplier) } toErrorOr
+        val minimumSummaryEntryId = Try { freeze.getOrElse("minimum-summary-entry-id", defaultMinimumSummaryEntryId) } toErrorOr
+        val debugLogging = Try { freeze.getOrElse("debug-logging", defaultDebugLogging)}  toErrorOr
 
-        for {
-          freezeScanConfig <- errorOrFreezeScanConfig.toEither
-          _ <- if (freezeScanConfig.maxInterval > freezeScanConfig.initialInterval) "".validNelCheck else "'max-interval' must be greater than or equal to 'initial-interval' in Carboniter 'freeze-scan' stanza".invalidNelCheck
-          _ <- if (freezeScanConfig.multiplier > 1) "".validNelCheck else "'multiplier' must be greater than 1 in Carboniter 'freeze-scan' stanza".invalidNelCheck
-        } yield freezeScanConfig
-      } else {
-        HybridCarboniteFreezeScanConfig().validNelCheck
-      }
+        val errorOrFreezingConfig: ErrorOr[MetadataFreezingConfig] = (initialInterval, maxInterval, multiplier, minimumSummaryEntryId, debugLogging) flatMapN {
+          case (i, m, x, s, d) =>
+            i match {
+              case f: FiniteDuration =>
+                for {
+                  _ <- if (s.exists(_ < 0)) "`metadata-freezing.minimum-summary-entry-id` must be greater than or equal to 0. Omit or set to 0 to allow all entries to be summarized.".invalidNel else "".validNel
+                  _ <- if (f > m) s"'max-interval' $m should be greater than or equal to finite 'initial-interval' $f".invalidNel else "".validNel
+                  _ <- if (x > 1) "".validNel else "`metadata-freezing.multiplier` must be greater than 1 in Carboniter 'metadata-freezing' stanza".invalidNel
```

Comment:
TOL if these validation steps were done as part of the initial parsing, I think you could take advantage of the "all errors at the same time" property of the applicative `flatMapN`. This way, you'll only get one error at a time.